### PR TITLE
Remove Fullname search parameter from status board product, application, and service. 

### PR DESCRIPTION
**Pull request by tinaafitz**
ocm-api-model changes: https://github.com/openshift-online/ocm-api-model/pull/872.

### CODE REVIEW

**Comment by tinaafitz** [comment]
Hi @smcavey, Please review. :-)

**Comment by smcavey** [comment]
Is there a model version change that should be associated with these changes?

**Comment by tinaafitz** [comment]
Hi @smcavey 
> Is there a model version change that should be associated with these changes?

Yes. I added a commit to update the version. :-)

**Comment by smcavey** [comment]
lgtm :)

**Comment by tinaafitz** [comment]
Hi @tzvatot, Can you review this MR when you have time?  Thank you so much. :-)

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.399
+- [OSDEV-1296] Remove StatusBoard `fullname` search parameter.
+- Model updated to version v0.0.352.
```
Comment (tzvatot) [inline CHANGES.md:8]: Bumping the model has a certain syntax for it, see other examples in this file. Also note that 0.0.352 includes other changes other than OSDEV-1296

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (tinaafitz) [reply CHANGES.md]: Hi @tzvatot, Thank you so much for the feedback. I modified the syntax and added an entry for Deprecating the `Notify` resource.  Is there anything else I should change?

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.399
+- Remove `StatusBoard` `fullname` search parameter.
+- Deprecate `Notify` resource.  
+- Update model version v0.0.352.
```
Comment (tzvatot) [inline CHANGES.md:9]: Please see example on how to document this: https://github.com/openshift-online/ocm-sdk-go/blob/main/CHANGES.md#01391

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (tzvatot) [reply CHANGES.md]: @tinaafitz the entry should be below the model update line. I've linked an example. Please see other example when documenting bump of a model version.

File: CHANGES.md [hunk 3]
```diff
@@ -3,6 +3,12 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.400
+- Update model version v0.0.352
+  - Remove `StatusBoard` `fullname` search parameter.
+- Update model version v0.0.351
```
Comment (tzvatot) [inline CHANGES.md:9]: There's no such version (https://github.com/openshift-online/ocm-api-model/releases). Please move the next line under v0.0.352

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 3 above)
Comment (tinaafitz) [reply CHANGES.md]: Thanks, @tzvatot.  I'm so sorry for being dense.  

I thought I had to account for the missing version here. https://github.com/openshift-online/ocm-api-model/blob/main/CHANGES.md.

**Review by tzvatot — APPROVED** [review]
LGTM

**Comment by tzvatot** [comment]
https://github.com/openshift-online/ocm-sdk-go/releases/tag/v0.1.400 created

**Pull request merged**